docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -15,42 +15,43 @@ For a better syntax validation component now is defined as a polymorphic type an
+. Replace Alias with Name. See  https://github.com/devfile/api/issues/9[corresponding issue].
 
-. Renaming dockerimage component type. See  https://github.com/che-incubator/devworkspace-api/issues/8[corresponding issue].
+. Renaming dockerimage component type. See  https://github.com/devfile/api/issues/8[corresponding issue].
 
-. Specify sources path for containers. See  https://github.com/che-incubator/devworkspace-api/issues/17[corresponding issue].
+. Specify sources path for containers. See  https://github.com/devfile/api/issues/17[corresponding issue].
 
-. Specify size of volume for component. See https://github.com/che-incubator/devworkspace-api/issues/14[corresponding issue].
+. Specify size of volume for component. See https://github.com/devfile/api/issues/14[corresponding issue].
 
-. Containers endpoints (routes/ingresses). See https://github.com/che-incubator/devworkspace-api/issues/33[corresponding issue].
+. Containers endpoints (routes/ingresses). See https://github.com/devfile/api/issues/33[corresponding issue].
 
 
 [role="_additional-resources"]
 .Additional resources
 
 
-* Component is a polymophic type. See https://github.com/che-incubator/devworkspace-api/issues/4[corresponding issue].
-* Shared Volumes Across Components. See https://github.com/che-incubator/devworkspace-api/issues/19[corresponding issue].
-* Out of Main Pod Compoenents. See  https://github.com/devfile/api/issues/48[corresponding issue].
-* Replace Alias with Name. See  https://github.com/che-incubator/devworkspace-api/issues/9[corresponding issue].
-* Renaming dockerimage component type. See  https://github.com/che-incubator/devworkspace-api/issues/8[corresponding issue].
-* Specify sources path for containers. See  https://github.com/che-incubator/devworkspace-api/issues/17[corresponding issue].
-* Specify size of volume for component. See https://github.com/che-incubator/devworkspace-api/issues/14[corresponding issue].
-* Containers endpoints (routes/ingresses). See https://github.com/che-incubator/devworkspace-api/issues/33[corresponding issue].
\ No newline at end of file
+* Component is a polymophic type. See https://github.com/devfile/api/issues/4[corresponding issue].
+* Shared Volumes Across Components. See https://github.com/devfile/api/issues/19[corresponding issue].
+* Out of Main Pod Components. See  https://github.com/devfile/api/issues/48[corresponding issue].
+* Replace Alias with Name. See  https://github.com/devfile/api/issues/9[corresponding issue].
+* Renaming dockerimage component type. See  https://github.com/devfile/api/issues/8[corresponding issue].
+* Specify sources path for containers. See  https://github.com/devfile/api/issues/17[corresponding issue].
+* Specify size of volume for component. See https://github.com/devfile/api/issues/14[corresponding issue].
+* Containers endpoints (routes/ingresses). See https://github.com/devfile/api/issues/27[corresponding issue].
```

Comment:
Is this block of links to issues needed? I'm not sure what value it adds to the content.

Reply:
Yes, we'll need to do serious revisions on this doc so it actually adheres to a procedure structure. We have issues open for language reviews, but not for procedure structural and organizational changes. 

Since it's essentially the same list of links twice, think we should remove the links after the code snippet and only include the links under `additional resources`?

Reply:
> We have issues open for language reviews, but not for procedure structural and organizational changes.

Could you please open issues for that then? Namely:

* Turning 'procedures' into real procedures (from the current form where it's just a list of links).
* Eliminating these redundant 'Additional resources' (I'd argue that these links to issues should not be featured anywhere in docs unless really pertinent).

Reply:
I'm resolving this conversation since we now have issues opened for the concerns we raised in this conversation.